blogposts/2025-01-24-2025-new-architecture-on-76-and-77.md

```diff
@@ -0,0 +1,91 @@
+
+In the Old Architecture (Paper), we relied on the Universal Windows Platform ([UWP](https://learn.microsoft.com/en-us/windows/uwp/)). With the New Architecture (Fabric), we’ve transitioned to the modern [Windows App SDK](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/). This upgrade allows us to leverage React Native’s cross-platform rendering logic while better integrating Windows-specific features.
+
+All New Architecture apps will now be Win32-based and built on the Windows App SDK, aligning with Microsoft’s latest app development recommendations. Meanwhile, Old Architecture apps will remain UWP-based, and it’s still possible to create and maintain them using legacy templates.
+
+While there are no immediate plans to deprecate Old Architecture, future investments are focused on the New Architecture. As React Native moves away from Old Architecture, React Native for Windows will follow, with clear migration guidance provided when the time comes.
+
+Have more questions? Read our [FAQ in our new architecture landing page](https://microsoft.github.io/react-native-windows/docs/new-architecture#faq)!
+
+## 🛠️ Steps on creating on new architecture application
+
+Let's dive into creating a new architecture application. You'll find the steps below are almost exactly the same as creating a old architecture application - with the only change being is specifying a new architecture template in step 4.
+
+1. Create a new React Native Project
+
+`npx --yes @react-native-community/cli@latest init <projectName> --version "latest"`
+
+2. Navigate into this newly created directory
+
+`cd <projectName>`
+
+3. Add React Native Windows to your project's dependencies
+
+`yarn add react-native-windows@^0.77.0`
+
+4. Initialize the React Native Windows native code and projects with the [new architecture templates](https://microsoft.github.io/react-native-windows/docs/init-windows-cli#templates)
+
+`yarn react-native init-windows --template cpp-app --overwrite`
+
+5. Run your RNW app as normal
+
+`npx react-native run-windows`
+
+Congrats, you’ve just created an RNW app using the new architecture! 🎉
+
+[new-fabric-app-2025](assets/2025-01-24-2025-new-architecture-on-76-and-77/new-fabric-app.gif)
+
+## 🚧 New Architecture Changes in 0.76 and 0.77
+
+### 📑 New Templates
+
+With the preview release of the new architecture, we're introducing brand-new templates! These templates are built to use the precompiled Microsoft.ReactNative NuGet packages by default, enabling faster build times. In the previous architecture, this experience was experimental and had limited support. However, in the new architecture, it has been the primary approach from day one.
+
+We now offer two new templates:
+
+cpp-app: For React Native Windows Applications (New Architecture, C++, Win32, Hermes)
```

Comment:
Recommend having a quick callout about how to use these templates, similar in format to the section above.